### PR TITLE
Equifax breach draft.

### PR DESCRIPTION
**Pull request by kmaida**
Needs design.

### CODE REVIEW

File: _posts/2017-09-08-equifax-data-breach.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## What Should I Do?
+
+You _can_ check to see if your data has been exposed by visiting [Equifax's 2017 Incident Potential Impact website](https://www.equifaxsecurity2017.com/potential-impact/) and clicking the "Check Potential Impact" button. However, before doing so, please make sure you read and understand the Terms of Service as well as the following article: [By signing up on Equifax’s help site, you risk giving up your legal rights](https://www.washingtonpost.com/news/the-switch/wp/2017/09/08/what-to-know-before-you-check-equifaxs-data-breach-website/?utm_term=.5f7b92ed1765). If you agree to the terms, you can fill in your last name and last six digits of your Social Security number to find out if your data has potentially been leaked and receive a date to come back to enroll in free credit monitoring for a year. The [Federal Trade Commission also has detailed instructions available here](https://www.consumer.ftc.gov/blog/2017/09/equifax-data-breach-what-do).
```
Comment (kukicado) [inline _posts/2017-09-08-equifax-data-breach.markdown]: you risk giving up your legal rights => you risk giving up your legal rights, particularly the right to participate in a class action lawsuit against Equifax.

WDYT?

File: _posts/2017-09-08-equifax-data-breach.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+In addition, you can monitor your own credit by requesting a free copy from each of the three major CRAs once a year. If requests are staggered between the agencies, you can check your credit once every four months. You can also [freeze your credit](http://clark.com/personal-finance-credit/credit-freeze-and-thaw-guide/) if you choose. If you do this, make sure you freeze it with all three major CRAs, not just Equifax.
+
+## Aside: Use Auth0 to Protect Your Users From Data Breaches
+
+Auth0 provides breached password detection that can be enabled simply and easily. The Auth0 security team maintains a database of third party breached credentials. This database is updated daily. On each login, user credentials are checked against the leaked passwords database. If a breach is suspected, the user is notified and cannot log in until they change the compromised password.
```
Comment (kukicado) [inline _posts/2017-09-08-equifax-data-breach.markdown]: i  would link "breached password detection" to the landing page for BPA here as well.

File: _posts/2017-09-08-equifax-data-breach.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+<video autoplay="" loop="" width="600">
+  <source src="https://cdn.auth0.com/blog/breach/lock-2.m4v">
+<img src="https://cdn.auth0.com/blog/breach/lock.gif">
+</video>
+
+Read about [Breached Passwords](https://auth0.com/breached-passwords) and [Anomaly Detection](https://auth0.com/docs/anomaly-detection) to learn more about how to protect users and their credentials with Auth0. You can also <a href="javascript:signup()">sign up for a free Auth0 account</a>!
```
Comment (kukicado) [inline _posts/2017-09-08-equifax-data-breach.markdown]: You can also ... => Sign up for a free Auth0 account and start protecting your users today.

WDYT?

File: _posts/2017-09-08-equifax-data-breach.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Another thing you can do is actively check your bank and credit card statements regularly. If you see any suspicious or unusual activity, report it immediately to your financial institution. Don't become complacent and stop checking a week or two after a breach. Staying aware and vigilant with your accounts is important at all times. [Unfortunately, large data breaches have occurred many times in the past](http://money.cnn.com/2017/09/07/technology/business/biggest-breaches-ever/index.html), and in the digital age, we should be as prepared as possible to face more in the future.
+
+In addition, you can monitor your own credit by requesting a free copy from each of the three major CRAs once a year. If requests are staggered between the agencies, you can check your credit once every four months. You can also [freeze your credit](http://clark.com/personal-finance-credit/credit-freeze-and-thaw-guide/) if you choose. If you do this, make sure you freeze it with all three major CRAs, not just Equifax.
```
Comment (kukicado) [inline _posts/2017-09-08-equifax-data-breach.markdown]: You may want to mention CreditKarma.com here. It's free for users and gets a report from both TransUnion and Equifax so they can consistently monitor their credit report.

File: _posts/2017-09-08-equifax-data-breach.markdown (hunk 1 above)
Comment (kmaida) [reply _posts/2017-09-08-equifax-data-breach.markdown]: 👍  added that to the end of the article title, outside the link.

File: _posts/2017-09-08-equifax-data-breach.markdown (hunk 4 above)
Comment (kmaida) [reply _posts/2017-09-08-equifax-data-breach.markdown]: Does this impact the frequency with which scores can be obtained though?

Comment (kukicado) [reply _posts/2017-09-08-equifax-data-breach.markdown]: Nope. With CreditKarma they update your score once a week. I've been using their service for a couple of years now and it's pretty solid. It doesn't ping the agencies either so you're not getting inquires on your account by using CK.

File: _posts/2017-09-08-equifax-data-breach.markdown (hunk 2 above)
Comment (kmaida) [reply _posts/2017-09-08-equifax-data-breach.markdown]: 👍  (Copy-pasted this section from another data breach article haha)

File: _posts/2017-09-08-equifax-data-breach.markdown (hunk 3 above)
Comment (kmaida) [reply _posts/2017-09-08-equifax-data-breach.markdown]: 👍

File: _posts/2017-09-08-equifax-data-breach.markdown (hunk 4 above)
Comment (kmaida) [reply _posts/2017-09-08-equifax-data-breach.markdown]: Okay cool. I added a link to it.

**Pull request merged**